forwarder/forwarder.go

```diff
@@ -0,0 +1,57 @@
+package forwarder
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Forwarder interface {
+	Forward(tid string, uuid string, collectionType string, reqBody []byte) (*ForwarderResponse, error)
```

Comment:
after much discussion (and flipping a virtual coin), we decided to try and remove pointer returns from most of the returns (at least the returns of the interface methods)